app/views/groups/new.html.erb

```diff
@@ -1,14 +1,19 @@
-<h1>New group</h1>
+<div class="create-title">
+<h2>Create New Group</h2>
+<h4>Fill in the blanks below to create your new group. All fields are required.</h4>
+</div>
   <%= form_for @group do |f| %>
-    Name: <%= f.text_field :name, required: true %>
-    Type: <%= f.select(:kind, @kinds) %>
-    Description: <%= f.text_field :description, required: true %>
-    Neighborhood: <%= f.select(:neighborhood_id, @neighborhoods) %>
-
+  <div class="create-section row">
+    Group Name <%= f.text_field :name, required: true, :placeholder => 'Example: Kensington Knitting Club' %>
+    Group Description <%= f.text_field :description, required: true, :placeholder => 'Example: This club is for...' %>
+    Group Cover Photo
     <div class="field">
-      <%= f.label :photo %>
+      <%= image_tag :photo %>
```

Comment:
Should have some logic to have the stock image when no photo is uploaded yet right? Remind me to get you this asset